frontend/src/app/containers/HomePage.js

```diff
@@ -0,0 +1,15 @@
+
+import React from 'react';
+
+import HomePageNoAddon from './HomePageNoAddon';
+import HomePageWithAddon from './HomePageWithAddon';
+
+export default class HomePage extends React.Component {
+  render() {
+    if (this.props.hasAddon) {
+      window.history.replaceState({}, '', '/experiments');
```

Comment:
Would be nice to have a test associated with this new logic. Also, `window.history.replaceState()` should be wrapped up in the App component and passed in as a prop [like other uses of `window`](https://github.com/mozilla/testpilot/blob/master/frontend/src/app/containers/App.js#L259). We've tried to keep direct access to browser APIs out of individual components, which should also make writing a test easier.